.github/workflows/ci.yaml

```diff
@@ -124,6 +124,7 @@ jobs:
         - 1.18.15
         - 1.19.7
         - 1.20.2
+        - 1.22.5
```

Comment:
remove 1.22.5 from this PR. It will be added from https://github.com/vmware-labs/service-bindings/pull/216

```suggestion

```